crates/wasmparser/src/validator/component_types.rs

```diff
@@ -1646,7 +1654,9 @@ impl ComponentDefinedType {
 
             ComponentDefinedType::Variant(ty) => ty.lower_gc(types, abi, options, offset, core),
 
-            ComponentDefinedType::List(ty) | ComponentDefinedType::FixedSizeList(ty, _) => {
+            ComponentDefinedType::List(ty)
+            | ComponentDefinedType::Map(ty, _)
```

Comment:
For this I think it would be best for `Map` to be routed to some sort of "not implemented" error and you can defer the work to someone fleshing out GC support in the future